Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG ARCH=amd64
-FROM --platform=linux/${ARCH} gcr.io/distroless/static-debian11
+FROM --platform=linux/${ARCH} gcr.io/distroless/static-debian12
```

Comment:
As per the warning in the github issue, this should not be merged yet as https://console.cloud.google.com/gcr/images/distroless/GLOBAL/static-debian12 does not yet exist.

Refer: https://github.com/GoogleContainerTools/distroless/issues/1337

Reply:
@jmhbnz  is it image does not exist globally or error in code

Reply:
Hey @SagarRawat24 - The image now exists globally, so these tests should now pass if you re trigger them as follows:

```bash
git reset --soft HEAD~1
git commit -s -a -m "Update to Debian 12 bookworm."
git push --force
```